### ISSUE TITLE
Normalizing images by default 

### ISSUE THREAD

**Issue by alono88** [pytorch/vision#1103]
I was wondering why image normalization is done in the model itself and not using the transforms in the dataloader. Wouldn't this behavior cause an issue of normalizing twice if the user assumes the normalization transform needs to be explicitly defined? many official tutorials, such as this tutorial on [finetuning models](https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html),  do define the normalization transforms explicitly. What am I missing?

https://github.com/pytorch/vision/blob/6693b2c6890a46734bbbfef9a3910db0d6b11c87/torchvision/models/detection/generalized_rcnn.py#L47

**Comment by fmassa** [comment]
This is a great question.

I'm thinking about moving part of the transforms away of the dataset, and into the model.

Indeed, different models might have different normalizations for the same dataset.
Also, if we want the transforms to be traceable (so that they work on C++), they need to leverage torch tensors, so putting it in the model is a logical thing to do.

I agree the current trends are to add `transforms` as being part of the dataset, but I'm experimenting with changing it, because it might have better trade-offs down the road (but still needs to be verified).

Let me know if you have more questions!
